challenges/prototypes.js

```diff
@@ -1,33 +1,44 @@
 /* ===== Prototype Practice ===== */
 
-// Task: You are to build a cuboid maker that can return values for a cuboid's volume or surface area. Cuboids are similar to cubes but do not have even sides. Follow the steps in order to accomplish this challenge.
+// Task: You are to build a cuboid maker that can return values for a cuboid's
+// volume or surface area. Cuboids are similar to cubes but do not have even sides.
+// Follow the steps in order to accomplish this challenge.
 
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
-
+function CuboidMaker (l, w, h) {
```

Comment:
This should just take in one parameter, `dimensions`, then you assign it such that `this.l = dimensions.l` and so on.